javascripts/govuk/show-hide-content.js

```diff
@@ -0,0 +1,150 @@
+(function (global) {
+  'use strict';
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+
+  function ShowHideContent() {
+    var self = this;
+
+    // Radio and Checkbox selectors
+    var selectors = {
+      namespace: 'ShowHideContent',
+      radio: '.block-label input[type="radio"]',
+      checkbox: '.block-label input[type="checkbox"]'
```

Comment:
@nickcolley For example, I've spotted you're also lacking a `role="region"` on the show-hide div and aren't passing `.focus()` when the content is toggled open/closed.

Just mentioned this to @gemmaleigh.

If you think I'm the right person to progress with this I'm happy to help?